css/style.css

```diff
@@ -1,12 +1,112 @@
-#navbar {
-    position: fixed;
-    top: 0;
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap");
+
+* {
+  font-family: "Open Sans", sans-serif;
+}
+
+html,
+body {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  background: #0799d5;
+}
+
+#formpage {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+form {
+  background-color: #3a469c;
+  border-radius: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+#title,
+#description,
+label {
+  color: white;
+}
+
+#title {
+  margin-top: 30px;
+  margin-bottom: 10px;
+}
+
+#description {
+  margin-top: 0px;
+  margin-bottom: 30px;
+}
+
+#name,
+#email,
+#number,
+#dropdown {
+  min-width: 400px;
+}
+
+#name,
+#email,
+#number,
+#dropdown,
+#textarea {
+  border: 3px solid #0799d5;
+}
+
+#name,
+#email,
+#number {
+  height: 25px;
+}
+
+#dropdown {
+  padding-top: 2px;
+  padding-bottom: 2px;
+  width: 100%;
+}
+
+label {
+  margin-bottom: 10px;
+}
+
+textarea {
+  resize: none;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.turn-info-fc {
+  padding: 5px;
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
 }
-#welcome-section {
-    height: 100vh;
+
+#button-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
 }
 
+button {
+  border: 1px solid #f8f9fa;
+  font-size: 14px;
+  height: 36px;
+  border-radius: 4px;
+  color: #3c4041;
+  background-color: #f8f9fa;
+  margin: 25px;
+}
 
-@media (min-width: 400px) {
```

Comment:
remove code that is not used to make code clean

Reply:
- [ ] Kindly make sure to deliver politely, and well-explained review comments so that students understand the point 🙏 😄 

_Kindly see the first comment for more clarification._

Reply:
- [ ] Dear @Mucyosoda, I think you should make the changes requested by the reviewer pointed out in the above comment. Maybe, thanks, is not the right place here. After this review, you should put the changes you make right below this comment to address this review comment I give you.